### DISCUSSION TITLE
Snackbar

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1126] [Proposals]
Buenas tardes.

@yceballost 

Estaba jugando con el componente Snackbar y noté una posible modificación.

La versión de Android cambia de tamaño en altura (de 48 a 52) cuando la opción "critical" está activa.

<img width="1792" alt="Captura de Tela 2023-03-14 às 15 14 14" src="https://user-images.githubusercontent.com/127137234/225099675-88bd702e-9ac7-4f13-b73c-af1b87f80441.png">

**Comment by yceballost** [comment]
😱 lo meto como issue! gracias!!
